repo/packages/P/portworx-elastic/600/resource.json

```diff
@@ -0,0 +1,60 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-master-28f8827.tar.gz",
+      "bootstrap-zip": "https://s3-us-west-1.amazonaws.com/px-dcos/dcos-commons/artifacts/0.40.5-1.3.7/bootstrap.zip",
+      "executor-zip": "https://s3-us-west-1.amazonaws.com/px-dcos/dcos-commons/artifacts/0.40.5-1.3.7/executor.zip",
```

Comment:
It looks like you're using a very old and outdated version of the SDK (latest is up to [0.57.2](https://github.com/mesosphere/dcos-commons/releases)). This loses the benefits of features such as region/zone awareness, bug fixes, UX improvements, SDK plan updates, logging, error handling, etc. 

Updating to a newer version will benefit not only running of the service but troubleshooting issues that may arise.

Reply:
We are in the process of updating the sdk as a separate effort since we have made changes to the sdk to support Portworx volumes.